Sources/SmileID/Classes/DocumentVerification/View/CaptureButton.swift

```diff
@@ -15,3 +15,88 @@ struct CaptureButton_Previews: PreviewProvider {
         CaptureButton {}
     }
 }
+
+import UIKit
+
+class TransparentCenterView: UIView {
```

Comment:
Can we move these to their own files? It does not make sense for them to be in a file called CaptureButton